contracts/minter/src/error.rs

```diff
@@ -42,6 +42,9 @@ pub enum ContractError {
     #[error("Invalid token id")]
     InvalidTokenId {},
 
+    #[error("AlreadyStarted")]
```

Comment:
This message could be improved. `Minting already started. Can not change start time`